### PR TITLE
Update Scala.js to 1.1.0

### PR DESCRIPTION
**Pull request by lolgab**
Another PR for you @cornerman :)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ scala:
 cache:
   yarn: true
   directories:
-    - $HOME/.ivy2/cache
+    - $HOME/.cache/
```
Comment (cornerman) [inline .travis.yml:11]: Why did you remove the `.ivy/cache`? I still have it on my pc - not sure about the travis containers.

**Comment by cornerman** [comment]
Awesome! Thank you and also for fixing unused warnings. Just had one question about the travis cache.

File: .travis.yml (hunk 1 above)
Comment (lolgab) [reply .travis.yml:11]: I removed it because sbt 1.3+ uses coursier which uses `~/.cache/coursier/v1` to store the cached artifacts.

Comment (cornerman) [reply .travis.yml:11]: Ah good point. thanks! did not think about that!
